Gemfile

```diff
@@ -51,6 +52,7 @@ group :development do
   gem 'listen', '~> 3.3'
   gem 'rack-mini-profiler', '~> 2.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
```

Comment:
we can remove this comment as right now it is not near Spring, and also we don't need it :)

Reply:
I have removed unnecessary comments and order gems alphabetically